src/Perl6/Compiler.nqp

```diff
@@ -174,18 +174,20 @@ Note that only boolean single-letter options may be bundled.
 The following environment variables are respected:
 
   RAKUDOLIB   Modify the module search path
-  PERL6LIB    Modify the module search path
+  RAKULIB     Modify the module search path
+  PERL6LIB    Modify the module search path # to be deprecated
   RAKUDO_HOME Override the path of the Rakudo runtime files
-  PERL6_HOME  Override the path of the Rakudo runtime files
+  RAKU_HOME   Override the path of the Rakudo runtime files
```

Comment:
`RAKU_HOME` should be removed. This is where `RAKUDO_HOME` is the right one as it is compiler specific, not language specific.

Reply:
Okay, but by that correct reasoning, we should also remove PERL6_HOME, correct?